package.json

```diff
@@ -6,7 +6,7 @@
   "private": true,
   "scripts": {
     "dev:simple":
-      "LOG_LEVEL=silly SERVER_SECRET=superSecret ts-node-dev --notify=false packages/tentales-example-simple/src/index.ts",
+      "LOG_LEVEL=silly ts-node-dev --notify=false packages/tentales-example-simple/src/index.ts",
```

Comment:
Should we maybe keep the secret here, to make encourage people to use it?

Reply:
While I agree that security-by-default is "a good thing"™, I also feel like the DX would be hampered if we had this on by default. I think another option might be to leave this off but fail with an error if you try to run unsecured with `NODE_ENV=production`?